packages/jest-haste-map/src/index.ts

```diff
@@ -739,13 +743,47 @@ class HasteMap extends EventEmitter {
     const crawlerOptions: CrawlerOptions = {
       computeSha1: options.computeSha1,
       data: hasteMap,
+      enableSymlinks: options.enableSymlinks,
       extensions: options.extensions,
       forceNodeFilesystemAPI: options.forceNodeFilesystemAPI,
       ignore,
       rootDir: options.rootDir,
       roots: options.roots,
     };
 
+    const watchmanConfigPath = path.join(options.rootDir, '.watchmanconfig');
+    if (existsSync(watchmanConfigPath)) {
+      try {
+        const configStr = readFileSync(watchmanConfigPath, 'utf8')
+          .toString()
+          .trim();
+        const watchmanConfig = configStr === '' ? {} : JSON.parse(configStr);
+        const watchSymlinks = Boolean(watchmanConfig['watch_symlinks']);
+
+        // Automatically enable symlink crawling in node crawler if watchman is
+        // set up to crawl symlinks.
+        if (watchSymlinks && !crawlerOptions.enableSymlinks) {
+          this._console.warn(
+            `jest-haste-map: watch_symlinks is enabled in .watchmanconfig ` +
+              `but --enableSymlinks was not passed to Jest as a flag. ` +
+              `  This will result in different behavior when watchman ` +
```

Comment:
```suggestion
              `This will result in different behavior when watchman ` +
```

Lots of extra spacing here, looks off

![image](https://user-images.githubusercontent.com/1404810/97689630-28672200-1a9c-11eb-94f6-5bdec0d2332e.png)